misk/src/main/kotlin/misk/security/keys/KeyManager.kt

```diff
@@ -0,0 +1,12 @@
+package misk.security.keys
+
+import okio.ByteString
+
+/** Handles encryption and decryption using keys stored in a key management service */
+interface KeyManager {
```

Comment:
This doesn't really manage keys. It uses keys to encrypt and decrypt, possibly misnamed?

Reply:
I think the most important feature is that this thing encapsulates the keys

Reply:
`KeyService`, as it is an interface to key services on GCP/AWS/etc?